src/models/form/howDoYouWantToPay.model.ts

```diff
@@ -0,0 +1,5 @@
+import PaymentType from '../dto/paymentType.enum';
+
+export default interface HowDoYouWantToPayModel {
+  paymentType: PaymentType
```

Comment:
I believe the forms all are suffixed with `FormModel`. Also `paymentType` should be optional as the user could submit the form without providing a paymentType, and the validator would catch this:
```
export default interface HowDoYouWantToPayFormModel {
  paymentType?: PaymentType
}
```

Reply:
Making this optional creates errors regarding typing. Do I now need to also add the usage of an Optional?

Reply:
That's where the `!` and `?` operators come in - usage of the `!` operator tells the compiler to assume that this value is present. This is safe to do once the form has been validated and no errors have been returned.